apiary.apib

```diff
@@ -428,35 +428,171 @@ Create a repository by providing its JSON representation.
+                "previous": null,
+                "results": [
+                    {
+                        "group_type": "administrators"
+                    },
+                    {
+                        "group_type": "curators"
+                    }
+                ]
+            }
+
+### Add Group to a Member [POST]
++ Request (application/json)
+
         {
-            "tbd": "tbd"
+            "group_type": "curators"
+        }
+
++ Response 201 (application/json)
+
+    + Headers
+
+            Location: /api/v1/repositories/physics-1/members/users/james/groups/curators/
+
+    + Body
+
+        {
+            "group_type": "curators"
         }
 
-## Members Collection [/repositories/{repo_slug}/members/{?role_id}]
+
+## Group for a Member [/repositories/{repo_slug}/members/users/{member_username}/groups/{group_type}/]
 
 + Parameters
-    + repo_slug: `physics-1` (number, required) - ID of the Repository
-    + role_id: ID of the role this member belongs to
+    + repo_slug: `physics-1` (string, required) - slug for the repository
+    + member_username: `patricia` (string, required) - username of the user in the system
+    + group_type: `curators` (string, required) - group type for the user in the system
 
+### View a Group For a Member [GET]
 
-### List All Members [GET]
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "group_type": "curators"
+            }
+
+### Delete a Group from a Member [DELETE]
+
++ Response 204 (application/json)
+
+## Members For a Group [/repositories/{repo_slug}/members/groups/{group_type}/users/]
+
++ Parameters
+    + repo_slug: `physics-1` (string, required) - slug for the repository
+    + group_type: `administrators` (string, required) - group type for the user in the system
+
+### View Members For a Group [GET]
 
 + Response 200 (application/json)
 
     + Body
 
-            [
-                {
-                    "tbd": "tbd"
-                }
-            ]
+            {
+                "count": 2,
+                "next": null,
+                "previous": null,
+                "results": [
+                    {
+                        "username": "james"
+                    },
+                    {
+                        "username": "patricia"
+                    }
+                ]
+            }
+
+### Add Member to a Group [POST]
++ Request (application/json)
+
+        {
+            "username": "james"
+        }
+
++ Response 201 (application/json)
+
+    + Headers
+
+            Location: /api/v1/repositories/physics-1/members/groups/administrators/users/james/
+
+    + Body
+
+        {
+            "username": "james"
+        }
+
+## Member for a Group [/repositories/{repo_slug}/members/groups/{group_type}/users/{member_username}/]
+
++ Parameters
+    + repo_slug: `physics-1` (string, required) - slug for the repository
+    + member_username: `james` (string, required) - username of the user in the system
```

Comment:
Can we swap this line with the next one so that they appear in the same order as the parameters in the URL?